xbmc/addons/AddonDatabase.cpp

```diff
@@ -149,7 +149,7 @@ void CAddonDatabase::CreateTables()
 
   CLog::Log(LOGINFO, "create repo table");
   m_pDS->exec("CREATE TABLE repo (id integer primary key, addonID text,"
-              "checksum text, lastcheck text, version text)\n");
+              "checksum text, lastcheck text, version text, nextcheck TEXT)\n");
```

Comment:
INTEGER

Reply:
It's a date/time, so shouldn't `TEXT` be fine? (same format as `lastcheck`)

Reply:
sure it works, but integer will prevent invalid values being written there. The header is an int as well

Reply:
this field does not contain the value of the header, but the date/time of the next check as determined when the repository was last updated (now() + header value)

Reply:
oh, ok then. Sorry for the noise